src/CMakeLists.txt

```diff
@@ -23,12 +23,6 @@ if(WIN32 AND BUILD_SHARED_LIBS)
   list(APPEND ONEMKL_BUILD_COPT "-Donemkl_EXPORTS")
 endif()
 
-# portBLAS backend variables must be accessible here to correctly
-# generate the config file.
-set(ENABLE_PORTBLAS_BACKEND_INTEL_CPU OFF CACHE INTERNAL "")
-set(ENABLE_PORTBLAS_BACKEND_INTEL_GPU OFF CACHE INTERNAL "")
-set(ENABLE_PORTBLAS_BACKEND_AMD_GPU OFF CACHE INTERNAL "")
-set(ENABLE_PORTBLAS_BACKEND_NVIDIA_GPU OFF CACHE INTERNAL "")
```

Comment:
Are you sure we can remove these lines? That seemed needed at the time. Is it working with portBLAS disabled and the minimum CMake version supported 3.13?

Reply:
yes I double checked with different version of cmake and all backends works correctly. Using cmake 3.13 I found other issues in `develop` branch. I'll open an issue for those soon.